templates/primary-secondary/README.md

```diff
@@ -30,32 +45,34 @@ Mental Models and Page Structure:
 
 If no nodes are assigned to the `footer` slot, the footer is hidden.
 
-**Note:** this template automatically includes `<header>`, `<main>`, `<aside>` and `<footer>` elements, so there's no need to include them inside the various slots.
+Note: this template automatically includes `<header>`, `<main>`, `<aside>` and `<footer>` elements, so there's no need to include them inside the various slots.
```

Comment:
It might make sense to move this and `If no nodes are assigned to the `footer` slot, the footer is hidden.` above it to above the demo since otherwise they end up pushed far below the demo tables